README.md

```diff
@@ -1 +1,157 @@
-ProgressView
+<H1 align="center">ProgressView</H1>
+
+<p align="center">
+<a href="https://cocoapods.org/pods/progress-view"><img alt="Version" src="https://img.shields.io/cocoapods/v/progress-view.svg?style=flat"></a> 
+<a href="https://github.com/Incetro/progress-view/blob/master/LICENSE"><img alt="Liscence" src="https://img.shields.io/cocoapods/l/progress-view.svg?style=flat"></a> 
+<a href="https://developer.apple.com/"><img alt="Platform" src="https://img.shields.io/badge/platform-iOS-green.svg"/></a> 
+<a href="https://developer.apple.com/swift"><img alt="Swift4.2" src="https://img.shields.io/badge/language-Swift5.0-orange.svg"/></a>
+</p>
+
+## Description
+
+This library is a custom progress view.
+ 
+- [Usage](#Usage)
+- [Requirements](#requirements)
+- [Communication](#communication)
+- [Installation](#installation)
+- [Authors](#license)
+
+## Usage <a name="Usage"></a>
+
+To use a *ProgressView*, all you will need to import `ProgressView` module into your swift file:
+
+```swift
+import ProgressView
+```
+
+There are two types of progress view in *ProgressView*, such as `ProgressBar` and `ProgressRing`. Both of them have similar properties and methods. Public properties are designed to customize the desired appearance, fill animation and property `progress` to get current progress value.
+
+```swift
+/// ProgressBar instance
+let progressBar = ProgressBar()
+
+/// The first gradient color
+progressBar.startColor = .red
+
+/// The second gradient color
+progressBar.endColor = .green
+
+/// The groove color in which the fillable ring resides
+progressBar.grooveColor = .blue
+
+/// Timing function of the progress fill animation
+progressBar.timingFunction = .easeInEaseOut
+
+/// Printing current progress value
+print(progressBar.progress) // outputs a Float value from 0 to 1
+
+/// Setting progress
+progressBar.setProgress(0.5, animated: true, completion: nil)
+```
+
+You can do the same with ProgressRing and something else, such as setting up the ring itself.
```

Comment:
You can do the same with ProgressRing and you can do a bit more, such as setting up the ring itself